picard/util/tags.py

```diff
@@ -196,3 +197,23 @@ def parse_comment_tag(name):  # noqa: E302
         desc = ''
 
     return lang, desc
+
+
+def parse_lyrics_tag(name):
```

Comment:
Generally agree on this, with one limitation: This PR should only add and use the `parse_subtag` function, but leave the current `parse_comment_tag` alone. If we do the small syntax changes I proposed this will be incompatible with the current syntax, and we need to consider how to handle the update (and only do that for Picard 3 anyway). So this should all go into a separate PR